inception-html-editor/src/main/java/de/tudarmstadt/ukp/inception/htmleditor/annotatorjs/resources/annotator-full.js

```diff
@@ -13,13 +12,14 @@
 
 //
 
+
 // Generated by CoffeeScript 1.6.3
```

Comment:
This file is, as the comment suggests, also generated. Since this is the only file in the pull request and no other instances of this smell appear in the code, I suggest to close this pull request.